scene/2d/navigation_obstacle_2d.h

```diff
@@ -31,15 +31,13 @@
 #ifndef NAVIGATION_OBSTACLE_2D_H
 #define NAVIGATION_OBSTACLE_2D_H
 
+#include "scene/2d/node_2d.h"
 #include "scene/main/node.h"
 
-class Navigation2D;
-
 class NavigationObstacle2D : public Node {
 	GDCLASS(NavigationObstacle2D, Node);
 
-	Navigation2D *navigation = nullptr;
-
+	Node2D *parent = nullptr;
```

Comment:
Probably worth renamed this to something else, because the name goes in conflict with the `Node` parent and can be miss leading: https://github.com/godotengine/godot/blob/master/scene/main/node.h#L276-L277

Reply:
Done ! :)